apps/src/netsim/NetSimSendPanel.js

```diff
@@ -365,6 +365,10 @@ NetSimSendPanel.prototype.onAddPacketButtonPress_ = function (jQueryEvent) {
   }
 
   this.addPacket_();
+
+  // Scroll to end of packet area
+  var scrollingArea = this.getBody().find('.send-panel-packets');
+  scrollingArea.animate({ scrollTop: scrollingArea[0].scrollHeight }, 'fast');
```

Comment:
I put the packet editors in a scroll area with a max-height, so when you add many packets it's nice to scroll to the newest packet.